docs/core-study-details.md

```diff
@@ -96,12 +96,23 @@ Per resource, the optional fields are as follows:
 - Encounter
```

Comment:
This file doesn't have any count table updates, because I didn't add count tables for the new resources - discuss? Are these resources worth having new count tables for?